cmd/thanos/compact.go

```diff
@@ -189,6 +188,11 @@ func runCompact(
 		return errors.Wrap(err, "create compactor")
 	}
 
+	readinessProber, err := metricHTTPListenGroup(g, logger, reg, httpBindAddr, component.Compact)
+	if err != nil {
+		return errors.Wrap(err, "create readiness prober")
```

Comment:
could we wrap errors everywhere or not at all in this method.

https://github.com/improbable-eng/thanos/pull/656/files#diff-b66fc35e7e48dadb672df4dead7b6bc5R159

^ has no wrapping

Reply:
Will do